README.md

```diff
@@ -7,21 +7,52 @@
 [![CLA assistant](https://cla-assistant.io/readme/badge/newrelic/newrelic-kubernetes-operator)](https://cla-assistant.io/newrelic/newrelic-kubernetes-operator)
 [![Release](https://img.shields.io/github/release/newrelic/newrelic-kubernetes-operator/all.svg)](https://github.com/newrelic/newrelic-kubernetes-operator/releases/latest)
 
-Operator to manage New Relic resources
+Operator to manage New Relic resources.
 
-# Development Prerequisites
+Currently enables management of Alert Policies and NRQL Alert Conditions.
 
-To get required tooling:
+# Quick start test drive from zero
 
+Get docker, kubectl, and kind installed
+``` bash
+brew cask install docker
+brew install kustomize kubernetes-cli kind
+```
 
+Create a test cluster
 
-```bash
-brew install kubectl kubebuilder kustomize
+``` bash
+kind create cluster --name newrelic
+kubectl cluster-info
+```
+
+Install cert-manager
+
+``` bash
+kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.15.0/cert-manager.yaml
 ```
 
+Install the operator in the test cluster.
 
-You will also want to install Docker for Mac and enable its built-in kubernetes cluster functionality.
+``` bash
+kustomize build https://newrelic.github.io/newrelic-kubernetes-operator/config/default/ \
```

Comment:
so this link still isn't valid for me, should this be something else?

Reply:
I think this is in the chicken-and-egg territory I've been talking about elsewhere—one of the things that has to be public and merged before we can confirm it works or not.

If this still doesn't work after merging the PR, then perhaps we follow-up with an update to declare the quickstart is still broken and move this work to something that comes after we get the build pipeline pushing assets ... make it part of the first release.

Reply:
It's what I mean in the PR description:
> Gonna just predict there will be a few follow-up PRs to get things working for real.